src/styles/blocks/main.scss

```diff
@@ -0,0 +1,26 @@
+.main {
+  padding: 50px 40px;
+  display: grid;
+  justify-content: center;
+  place-items: center;
+  grid-template-columns: repeat(1, 200px);
+  gap: 46px 48px;
+
+  @media (min-width: 488px) {
+    & {
+      grid-template-columns: repeat(2, 200px);
```

Comment:
Check the design requirements for the minimum width of 488px to ensure that the number of columns and their widths are set correctly. If the design requires a different layout, adjust the grid-template-columns property accordingly.